utils/python/CIME/utils.py

```diff
@@ -149,7 +150,7 @@ def run_cmd(cmd, ok_to_fail=False, input_str=None, from_dir=None, verbose=None,
     if (arg_stderr is _hack):
         arg_stderr = subprocess.PIPE
 
-    logging.info("RUN: %s" % cmd)
```

Comment:
If you look above, this function takes a verbose flag. The idea is, some shell commands are so important that you always want to see "verbose" information about them. In most cases, the shell commands are not interesting and you don't want to see anything with default logging.